frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/vulnerabilities/VulnerabilitiesPortlet.java

```diff
@@ -92,11 +144,11 @@ private void shortenTimeStampsToDates(List<Vulnerability> vulnerabilities){
         });
     }
 
-    private String getDateFromFormattedTimeStamp(String formattedTimeStamp){
-        return formattedTimeStamp.substring(0,10);
+    private String getDateFromFormattedTimeStamp(String formattedTimeStamp) {
+        return formattedTimeStamp.substring(0, 10);
```

Comment:
Magic constant

Reply:
In a magic function, I don't think it brings anything to extract it in this case :)